### PR TITLE
[Grupa 1] PD3 Jamroży Bartosz

### PR DESCRIPTION
**Pull request by jamrozyb**

### CODE REVIEW

File: Prace_domowe/Praca_domowa3/Jamrozy_Bartosz/pd3_australian_weather.ipynb [hunk 1]
```diff
@@ -0,0 +1,493 @@
+{
```
Comment (ptomaszewska) [inline Prace_domowe/Praca_domowa3/Jamrozy_Bartosz/pd3_australian_weather.ipynb:1]: <p>Zabrakło krótkiego EDA aby wiedzieć czy nie należy dokonać inżynierii cech. W przypadku przeprowadzenia analiz okazałoby się, że zmienna Cloud3pm według dokumentacji powinna przyjmować wartości z zakresu 0-8, a raz w ramce danych pojawia się wartość 9.</p><p>Absolutne minimum to sprawdzenie balansu klas w zmiennej, która będzie targetem. Taka analiza implikuje czy accuracy jest dobrą miarą - dobrze, że na późniejszym etapie to zweryfikowano :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (ptomaszewska) [inline Prace_domowe/Praca_domowa3/Jamrozy_Bartosz/pd3_australian_weather.ipynb:1]: <p>Z uwagi na niezbalansowanie klas, warto byłoby zastosować parametr stratify w train_test_split</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (ptomaszewska) [inline Prace_domowe/Praca_domowa3/Jamrozy_Bartosz/pd3_australian_weather.ipynb:1]: <p>dobrze, że wyjaśniono parametry modeli, które ustawiono</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (ptomaszewska) [inline Prace_domowe/Praca_domowa3/Jamrozy_Bartosz/pd3_australian_weather.ipynb:1]: <p>Bardzo dobrze, że to sprawdzono!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (ptomaszewska) [inline Prace_domowe/Praca_domowa3/Jamrozy_Bartosz/pd3_australian_weather.ipynb:1]: <p><strong>W przypadku gdy klasy są niezbalansowane, accuracy nie jest najlepszą miarą.</strong></p><p>Dobrze, że do oceny wykorzystano też recall.</p><p> </p><p>Podsumowanie:</p><ul><li>Dobrze wykonana praca domowa.</li><li>Szkoda, że nie użyto kroswalidacji.</li><li>Można było jeszcze dodać confusion matrix.</li></ul><p> </p><p>Praca zostanie "zmergowana", gdy minie termin oddania PD przez inne grupy.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>
